schema.graphql

```diff
@@ -225,24 +225,51 @@ type AccruedInterestWithdrawalChange @entity {
   timestamp: BigInt!
 }
 
-type TTokenStatus @entity {
+####################################################################################
+#
+# It contains the last tToken holder balances.
+# Each row represents the current holder balance for a given tToken.
```

Comment:
holder = lender?

Reply:
Technically, the entity is to track the holder balances (including the lenders)